src/redux/notes/selectors.js

```diff
@@ -0,0 +1,6 @@
+import { createSelector } from 'reselect';
+
+export const getNotesSelector = createSelector(
+  state => state.notes,
+  items => items
+);
```

Comment:
since we are just using this selector in only 1 component, delete this file and create the selector directly in the component